engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Error.java

```diff
@@ -226,6 +229,10 @@ public ForbiddenOperation getForbiddenOperation() {
     return forbiddenOperation;
   }
 
+  public Atom makeNotImplemented(String operation) {
```

Comment:
Why are we introducing `NotImplemented` type?

Reply:
This was requested by @jdunkerley

Reply:
Realized there is `Unimplemted` so I changed it to that.